tests/methods/avas-8/input.dat

```diff
@@ -0,0 +1,114 @@
+# Test the pi orbital selection of AOSubspace on overlapped planes C20 molecule
+
+import forte
+
+refscf = -756.50604655444420
+reffci = -113.911667467206257
+
+molecule C20{
+# grabed from https://nanotube.msu.edu/fullerene/fullerene.php?C=20
+# C1 symmetry due to Jahn-Teller distortion
```

Comment:
To remind people state that fullerene molecule C20 is in cage form and usually has an icosahedral geometry with a Ih point group

Reply:
I think the C20 cage is usually in C1 but mentioning this is a cage is a good point.